### PR TITLE
Update readme

### LINKED ISSUE #53: Reflect in the documentation recent changes in bridge and contract deployment procedure
Please update the README.md file to reflect changes made under:
* https://github.com/poanetwork/poa-parity-bridge-contracts/pull/23 - to add management ownership with multisig tools
* https://github.com/poanetwork/gnosis-deploy/commit/d05d7cb6b4294b13ef89d85aa3b41dd6d3bfdca6 - to introduce multisig tool
* https://github.com/poanetwork/poa-parity-bridge-contracts/pull/21 - to change validator configuration in `.env` file
* https://github.com/poanetwork/parity-bridge/pull/46 - to add RPC support 

Also I think it is not necessary to install and configure parity binary any more. So, it could be removed from documentation also.

### PR DESCRIPTION
**Pull request by rstormsf**
@akolotov 
please review
@igorbarinov 

closes #53

### CODE REVIEW

**Comment by rstormsf** [comment]
Deploy preview for *poabridge* ready!

Built with commit 0e2067164dec74918ae760cb363a39aefd409d42

https://deploy-preview-55--poabridge.netlify.com

**Comment by akolotov** [comment]
@rstormsf have you seen my comments? I see that they were ignored with no reason.

File: README.md [hunk 1]
```diff
@@ -307,37 +225,38 @@ touch ForeignBridge_bytecode.bin
...
-#Full  IPC path to your HOME node
-ipc = "/Users/USERNAME/sokol-kovan-bridge/sokol-node/sokol-datadir/jsonrpc.ipc"
-# How many block confirmation to wait to send a tx
+account = "0xETH_ACCOUNT_VALIDATOR_SOKOL"
 required_confirmations = 0
+rpc_host = "https://sokol.poa.network"
+rpc_port = 443
+password = "password.txt"
 
 [home.contract]
```
[the remark this answers is not in the document]
Comment (rstormsf) [reply README.md]: why?

File: README.md [hunk 2]
```diff
@@ -307,37 +225,38 @@ touch ForeignBridge_bytecode.bin
...
 [authorities]
 accounts = [
-  "0xETH_ACCOUNT_VALIDATOR_KOVAN"
+  "0x..",
+  "0x.."
 ]
 required_signatures = 1
-
 [transactions]
 home_deploy = { gas = 3000000, gas_price = 1000000000 }
```
[the remark this answers is not in the document]
Comment (rstormsf) [reply README.md]: what not needed anymore? can u be more clear

File: README.md [hunk 3]
```diff
@@ -348,7 +267,7 @@ withdraw_confirm = { gas = 3000000, gas_price = 1000000000 }
 11. Create db.toml file  
 `nano db.toml`  
 Insert addresses from  
-`cat poa-parity-bridge-contracts/deploy/bridgeDeploymentResults.json | jq 'del(.[].bytecode)'`  
+`cat poa-bridge-contracts/deploy/bridgeDeploymentResults.json`  
 ```yaml
 home_contract_address = "0xE46...E0a" #YOUR HOME CONTRACT ADDRESS THAT YOU DEPLOYED IN STEP#3 
 foreign_contract_address = "0x90...3eA" #YOUR FOREIGN CONTRACT ADDRESS THAT YOU DEPLOYED IN STEP#4
```
[the remark this answers is not in the document]
Comment (rstormsf) [reply README.md]: why

[the remark this answers is not in the document]
Comment (rstormsf) [reply README.md]: I don't understand what you mean

**Comment by rstormsf** [comment]
@akolotov it was mistake because Denis PR broke deployment and my  previous commit somehow automatically merged PR.

**Comment by rstormsf** [comment]
https://github.com/poanetwork/bridge-ui/commit/20d938ac3247b13bb53000b9bd5d6cf49f6535bd

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (akolotov) [reply README.md]: The bridge binary was modified in recent commits to not deploy contracts and that's why `[home.contract]` and `[foreign.contract]` sections are not used any more in the config file.

**Comment by rstormsf** [comment]
@akolotov 
second of all, your comments are not clear. Please provide more specific feedback.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (akolotov) [reply README.md]: The bridge binary was modified in recent commits to not deploy contracts and that's why `home_deploy` and `foreign_deploy` parameters are not used any more in the config file.

**Comment by rstormsf** [comment]
@akolotov can u submit new  PR to update README?

**Pull request merged**